citi-onboarding/Aqua-Jr
# Feature/footer

## What I did
- Created the frontend for the Footer

## How to test
- Firstly, all changes made locally on your branch should be committed
- Now, access the branch through your terminal:
```
   git checkout develop
   git pull
   git checkout feature/footer
```
- To continue the testing process, you should now open a second terminal, or, if you're using the VSCode one, just slipt your current one
- On one terminal you should type the following commands to access the **client folder** and run **yarn start**

```
    cd client
    yarn start
```
- If all the previous steps were followed correctly you should now have a blank localhost300 page open on your server
- Moving on, go to your second terminal and access the **server folder**, where you should run  either **node index.js** or **nodemon index.js** 

```
    cd server
    nodemon index.js
```
- Now you're all set and should be able to view the website footer on your **localhost3000** page. To properly test it, try changing both the height and width of the page and check if the footer's responsiveness is fully working.
